charts/keycloakx/values.yaml

```diff
@@ -500,9 +500,9 @@ test:
   enabled: false
   image:
     # The image for the test Pod
-    repository: docker.io/unguiculus/docker-python3-phantomjs-selenium
+    repository: docker.io/joyzoursky/python-chromedriver
```

Comment:
The project / image does not seem to be very "maintained". But, as we replace an ever less maintained image, this is fine for now..

Reply:
I agree, but it seems that there is no image that continues to be maintained, so it would be better to createtest image in the repository in the future.